### ISSUE TITLE
Please update the compat levels in d/compat

### ISSUE THREAD

**Issue by cpaelzer** [CanonicalLtd/probert#68]
Hi,
found while checking probert for a MIR I wondered about the slightly outdated compat level.
I'd ask to at least go up to v11 and as usual check the compat levels in [debhelper](https://manpages.debian.org/testing/debhelper/debhelper.7.en.html) to adapt anything if needed.

There is a tradeoff to be made thou. If you only upload probert as-is to newer releases then you'd usually go as new as possible. But if you intend to later on upload the same (backport) versions in older releases you can only rely on what the debhelper there provides.

**Comment by xnox** [comment]
xenial is 9, xenial-backports is 10, bionic is 11, eoan is 12.

However, i thought we use different branches for uploads to different series, so should be able to bump bionic+ to 11 I guess?

**Comment by cpaelzer** [comment]
yeah you are right, even if new are uploaded with equal content to all releases since probert is >=Bionic then level 11 should still work without too much backport hassle.
